src/sentry/api/serializers/models/project.py

```diff
@@ -81,7 +80,7 @@
 }
 
 
-def _get_team_memberships(team_list: Sequence[Team], user: User) -> Iterable[int]:
+def _get_team_memberships(team_list: Sequence[int], user: User) -> Iterable[OrganizationMemberTeam]:
```

Comment:
Is there a benefit to changing the lookup used here to be `team_id__in` instead of `team_in`?